src/zenml/steps/step_interfaces/__init__.py

```diff
@@ -1,42 +0,0 @@
-#  Copyright (c) ZenML GmbH 2021. All Rights Reserved.
```

Comment:
I'm personally very happy to see these gone, but maybe @bcdurak or @htahir1 who wanted these have some good reasons to keep them?

Reply:
Same here. I do not see any reason why we should keep them.

Reply:
lets get rid of em